tests/functional/vault/test_shares.py

```diff
@@ -187,12 +187,6 @@ def test_transferFrom(accounts, token, vault, fn_isolation):
     with brownie.reverts():
         vault.transferFrom(a, b, vault.balanceOf(a) // 2, {"from": c})
 
-    # Show that approval ordering protection works
```

Comment:
if the change was redundant, why did you need to remove this part of the test?

Reply:
wrong choice of words, I meant overly onerous